### ISSUE TITLE
No abilities in web GUI

### ISSUE THREAD

**Issue by sc0ttex** [mitre/caldera#327]
I saw there is [issue#159](https://github.com/mitre/caldera/issues/159) closed, but i do not see abilities in web GUI.
Repo cloned yesterday.
I checked and all abilities seems to have description (as it sounded to be the problem referred in [issue#159](https://github.com/mitre/caldera/issues/159)).

I add a quick question: is it not possible in this new version to create an adversary via GUI selecting abilities as in older videos on YouTube? In adversary "tab" it is only possible to select existing adversaries.

Thanks

![image](https://user-images.githubusercontent.com/17064432/61874905-cb201080-aee9-11e9-8976-547501edae69.png)

**Comment by xenoscr** [comment]
Having the same issue.

**Comment by Doron-droid** [comment]
Facing the same problem with the latest version.

**Comment by privateducky** [comment]
agreed. I took the adversary updating out of the bleeding-edge code to revise it (between 2.2 and 2.3 versions). There were a few issues/confusions around how it worked and how the data persisted, which I've been aiming to resolve.

I just now pushed in the revised adversary creation process. I'll be pushing one more change in the next day or two, which will clear up any confusion around how a saved adversary (from the GUI) is persisted in the database. 

In the meantime, let me know your thoughts on the (completely fresh) GUI code! Remember-- since it's bleeding-edge code, do a full git clone (recursive) to get the updates correctly. I'll update here once I push the second PR in the next 1-2 days.
